core/sys/darwin/xnu_system_call_wrappers.odin

```diff
@@ -337,7 +337,7 @@ syscall_ftruncate :: #force_inline proc "contextless" (fd: c.int, length: off_t)
 	return cast(c.int)intrinsics.syscall(unix_offset_syscall(.ftruncate), uintptr(fd), uintptr(length))
 }
 
-syscall_sysctl :: #force_inline proc "contextless" (name: ^c.int, namelen: c.uint, oldp: rawptr, oldlenp: ^i64, newp: ^i8, newlen: i64) -> c.int {
+syscall_sysctl :: #force_inline proc "contextless" (name: [^]c.int, namelen: c.size_t, oldp: rawptr, oldlenp: ^c.size_t, newp: rawptr, newlen: c.size_t) -> c.int {
```

Comment:
`c.int` the ints are constants that make up a path to the value in question.

You can pass `[ CTL_KERN, CTL_MAXPROC ]` for example, this would correspond to passing `"kern.maxproc"` in `sysctlbyname` (which just does a lookup/mapping for you).

The constants do not seem to relate to codepoints.